### ISSUE TITLE
[Nala]: Change default theme temperature

### ISSUE THREAD

**Issue by fallaciousreasoning** [brave/brave-browser#40752]
Since we added the Nala color provider, we've been using a slightly warmer neutral tone than previously, as we've inherited the default one from Chromium.

We should override the Baseline palette with the Nala tones.

Test Plan:
1. Open an instance of Brave from before and after the fix. Line them up next to each other in light mode

The colors should be ever so slightly bluer after this change, and ever so slightly redder before it. It's pretty subtle:

After is above, before below:
![image](https://github.com/user-attachments/assets/5c0cc1eb-8716-4422-bb12-60e9281fb965)

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.71.105 Chromium: 129.0.6668.89 (Official Build) beta (x86_64)
-- | --
Revision | f69ba167992437799f1dbcce81e90ba501a3d63e
OS | macOS Version 14.7 (Build 23H124)
```

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.71.110 Chromium: 129.0.6668.100 (Official Build) beta (64-bit)
-- | --
Revision | 46cd0b1af7127687895fcaceff93dd80e29e6f2b
OS | Windows 11 Version 23H2 (Build 22631.4317)
```

Verified the issue from STR from the description in light mode and compared with `1.71.46` (before the fix) with 1.71.110 (after the fix)
Confirmed the color shown as expected 

horizontal view | vertical view 
----- | -----
![Image](https://github.com/user-attachments/assets/78b76e09-23c3-41ae-a3d3-e0bccb14be3e)|![Image](https://github.com/user-attachments/assets/bb5fa324-c259-4f0a-a75b-2da6aeafb0ef)

**Comment by GeetaSarvadnya** [comment]
Verified with

```
Brave	1.71.112 Chromium: 130.0.6723.44 (Official Build) (64-bit) 
Revision	7d51df656b247f9432ee714a6d160142a1e11c13
OS	Linux
```

Fix landed in 1.71.47, so grabbed 1.71.46 for comparison from https://github.com/brave/brave-browser/releases/tag/v1.71.46.

Compared coloring in 1.71.46 (Nightly) to 1.71.112 (RC) for both light and dark themes:

Light | Dark
--- | ---
![Image](https://github.com/user-attachments/assets/035146f8-b872-40c0-b448-5d30c845352f) | ![Image](https://github.com/user-attachments/assets/ab93799d-e706-4d86-bd4f-93d60ade6799)
![Image](https://github.com/user-attachments/assets/0151b69c-47c4-4e08-ac5a-f99feede464e) | ![Image](https://github.com/user-attachments/assets/89f6c891-4af6-444c-89c9-dbf54c6a06fb)
